lib/cli.ts

```diff
@@ -1,6 +1,6 @@
 import * as args from 'args'
 import * as path from 'path'
-import {ELogLevel} from './log'
+import { ELogLevel } from './log'
```

Comment:
Just curious - why did this change occur? Did your IDE use global prettier settings by any chance? Local settings seem to include: `"bracketSpacing": false,`